tests/integration/support/index.js

```diff
@@ -1,21 +0,0 @@
-// ***********************************************************
-// This example support/index.js is processed and
-// loaded automatically before your test files.
-//
-// This is a great place to put global configuration and
-// behavior that modifies Cypress.
-//
-// You can change the location of this file or turn off
-// automatically serving support files with the
-// 'supportFile' configuration option.
-//
-// You can read more here:
-// https://on.cypress.io/configuration
-// ***********************************************************
-
-// Import commands.js using ES2015 syntax:
-import "./commands";
-import "@cypress/code-coverage/support";
```

Comment:
So, from what I recall, there is no support for coverage anymore right?

Reply:
Yes. TestCafe doesn't really have a built-in mechanism yet to calculate coverage. See the [discussion thread](https://github.com/DevExpress/testcafe/issues/2778).

When I googled there seem to be some tools with their own browser capabilities. I think we can consider them later if really needed. That may mean moving away from BrowserStack (or having another browser service in addition to it).
https://www.lambdatest.com/blog/testcafe-lambdatest-test-coverage/